build/linux/installer/datafiles/base_container.data

```diff
@@ -213,11 +208,14 @@ MAINTAINER:              'Microsoft Corporation'
 /opt/td-agent-bit/bin;                      755; root; root;sysdir
 /etc/telegraf;                              755; root; root;sysdir
 
-/opt/microsoft/omsagent/plugin/lib;												755; root; root; sysdir
-/opt/microsoft/omsagent/plugin/lib/application_insights;                        755; root; root; sysdir
-/opt/microsoft/omsagent/plugin/lib/application_insights/channel;                755; root; root; sysdir
-/opt/microsoft/omsagent/plugin/lib/application_insights/channel/contracts;      755; root; root; sysdir
-/opt/microsoft/omsagent/plugin/lib/application_insights/rack;                   755; root; root; sysdir
+/etc/fluent;                                                         755; root; root; sysdir
```

Comment:
What is the rationale behind changing just the plugin path and keeping the old paths for all the logs?

Reply:
> What is the rationale behind changing just the plugin path and keeping the old paths for all the logs?

> plugin path changed because of the omsagent removal and log path also changed. When you say "old paths for all the logs", can you please point me which log paths you are referring here?

Reply:
only change with regards to log paths for the ones which are using /var/opt/microsoft/**omsagent**/log path. This change because of the omsagent removal. These will be moved under /var/opt/microsoft/dockercim-prov/log path.